sagafalabella/sagafalabella/spiders/falabella.py

```diff
@@ -0,0 +1,59 @@
+# -*- coding: utf-8 -*-
+import scrapy
+from urllib.parse import quote_plus
+import json
+from scrapy import Request
+
+class FalabellaSpider(scrapy.Spider):
+    name = 'falabella'
+    allowed_domains = ['falabella.com.pe/falabella-pe']
+    start_urls = ['http://falabella.com.pe/falabella-pe']
+
+    def parse(self, response):
+        # we are not interested in some anchors give us url to all categories
+        categories = response.xpath(
+            '//ul[@class="fb-masthead__grandchild-links"]/li/a[not(contains(text(),"Ver todo"))]/@href').extract()
+        for categorie in categories:
+            categorie_url = response.urljoin(categorie)
+            yield Request(categorie_url, callback=self.parse_categorie, dont_filter=True)
+
+    def parse_categorie(self, response):
+        
+        if len(response.xpath('//head/link[@rel="next"]/@href').extract()):
```

Comment:
`extract` could be a non-list element, you could change it to just `response.xpath('//head/link[@rel="next"]/@href').extract()`, as it would work the same:

```
In [3]: lelelist = []                                                                                                                                                                                               

In [4]:                                                                                                                                                                                                             

In [4]: if lelelist: 
   ...:     print('lele') 
   ...:                                                                                                                                                                                                             

In [5]: if len(lelelist): 
   ...:     print('lele') 
   ...:  
```